idaes/models/unit_models/heat_exchanger_1D.py

```diff
@@ -643,77 +596,116 @@ def initialize_build(
         opt = get_solver(solver, optarg)
 
         # ---------------------------------------------------------------------
-        # Initialize shell block
-        flags_shell = self.shell.initialize(
+        # Get length and area values
+        if self.area.fixed:
+            # Most likely case
+            self.cold_side.area.set_value(self.area)
+        elif self.cold_side.area.fixed:
+            # This would be unusual, but check
+            self.area.set_value(self.cold_side.area)
```

Comment:
Wait, I thought `ControlVolume1D.area` was cross sectional area. Also, `cold_side.area` gets fixed as part of building the model. Is it unfixed anywhere?

Reply:
Missed this (I originally conflated cross-sectional area and heat transfer area). This can be removed.